### ISSUE TITLE
Server document example missing import statment.

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#5130]
|<img alt="axelilly's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| axelilly reported|
|-|-|
|Trac ID|trac#5130|
|Type|enhancement|
|Created|2011-06-01 20:41:44Z|

The server document:
http://twistedmatrix.com/documents/current/core/howto/servers.html

Under the headline:
Using the Protocol

Is missing the following import statements:
from twisted.internet.protocol import Protocol
from twisted.internet.protocol import Factory

<details><summary>Searchable metadata</summary>

```
trac-id__5130 5130
type__enhancement enhancement
reporter__axelilly axelilly
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__duplicate duplicate
component__core core
keywords__ 
time__1306960904000000 1306960904000000
changetime__1306967585000000 1306967585000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="axelilly's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|axelilly set status to `closed`|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `reopened`|
|-|-|

This was fixed in r31670 for [#4761](https://github.com/twisted/twisted/issues/4761).

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)|@exarkun set status to `closed`|
|-|-|
